tools/local-snapshot-runner-app/src/test/localSnapshots/localSnapshot2.json

```diff
@@ -0,0 +1 @@
ddress\":\"4cdd4c7a-180f-4ec0-8057-8eff7ba81c7d\",\"contents\":{\"pos1\":59,\"seg\":{\"text\":\"i\",\"props\":{\"attribution\":{\"id\":\"sampleemail@microsoft.com\",\"name\":\"John Doe\",\"email\":\"sampleemail@microsoft.com\",\"timestamp\":1652216400000}}},\"type\":0}},\"type\":\"op\"}}}","metadata":{},"minimumSequenceNumber":64,"referenceSequenceNumber":65,"sequenceNumber":68,"term":1,"timestamp":1652216638475,"type":"op"}},{"sequenceNumber":69,"op":{"clientId":"5d281acb-967d-45e4-ae57-2a7734e38fe4","clientSequenceNumber":56,"contents":"{\"type\":\"component\",\"contents\":{\"address\":\"00ee3ad1-7b9f-4361-b477-a449de0d3f66\",\"contents\":{\"content\":{\"address\":\"4cdd4c7a-180f-4ec0-8057-8eff7ba81c7d\",\"contents\":{\"pos1\":60,\"seg\":{\"text\":\"s\",\"props\":{\"attribution\":{\"id\":\"sampleemail@microsoft.com\",\"name\":\"John Doe\",\"email\":\"sampleemail@microsoft.com\",\"timestamp\":1652216400000}}},\"type\":0}},\"type\":\"op\"}}}","metadata":{},"minimumSequenceNumber":64,"referenceSequenceNumber":67,"sequenceNumber":69,"term":1,"timestamp":1652216638558,"type":"op"}},{"sequenceNumber":70,"op":{"clientId":"5d281acb-967d-45e4-ae57-2a7734e38fe4","clientSequenceNumber":57,"contents":"{\"type\":\"component\",\"contents\":{\"address\":\"00ee3ad1-7b9f-4361-b477-a449de0d3f66\",\"contents\":{\"content\":{\"address\":\"4cdd4c7a-180f-4ec0-8057-8eff7ba81c7d\",\"contents\":{\"pos1\":61,\"seg\":{\"text\":\")\",\"props\":{\"attribution\":{\"id\":\"sampleemail@microsoft.com\",\"name\":\"John Doe\",\"email\":\"sampleemail@microsoft.com\",\"timestamp\":1652216400000}}},\"type\":0}},\"type\":\"op\"}}}","metadata":{},"minimumSequenceNumber":64,"referenceSequenceNumber":69,"sequenceNumber":70,"term":1,"timestamp":1652216638742,"type":"op"}},{"sequenceNumber":71,"op":{"clientId":"7cfd6ca4-c526-4cf3-9ae1-135e04727e59","clientSequenceNumber":4,"contents":"null","minimumSequenceNumber":69,"referenceSequenceNumber":70,"sequenceNumber":71,"term":1,"timestamp":1652216640394,"type":"noop"}},{"sequenceNumber":72,"op":{"clientId":"5d281acb-967d-45e4-ae57-2a7734e38fe4","clientSequenceNumber":58,"contents":"null","minimumSequenceNumber":70,"referenceSequenceNumber":71,"sequenceNumber":72,"term":1,"timestamp":1652216640861,"type":"noop"}},{"sequenceNumber":73,"op":{"clientId":null,"clientSequenceNumber":-1,"contents":null,"minimumSequenceNumber":70,"referenceSequenceNumber":-1,"sequenceNumber":73,"term":1,"timestamp":1652216642694,"type":"leave","data":"\"5d281acb-967d-45e4-ae57-2a7734e38fe4\""}},{"sequenceNumber":74,"op":{"clientId":null,"clientSequenceNumber":-1,"contents":null,"minimumSequenceNumber":74,"referenceSequenceNumber":-1,"sequenceNumber":74,"term":1,"timestamp":1652216642698,"type":"leave","data":"\"7cfd6ca4-c526-4cf3-9ae1-135e04727e59\""}},{"sequenceNumber":75,"op":{"clientId":null,"clientSequenceNumber":-1,"contents":null,"minimumSequenceNumber":75,"referenceSequenceNumber":75,"sequenceNumber":75,"term":1,"timestamp":1652216642760,"type":"noClient"}}]}
```

Comment:
Your other file is nicely formatted for human readability - it would be great for this one to be formatted that way too

Reply:
can we leverage the test data here?

Reply:
https://github.com/microsoft/FluidFrameworkTestData

Reply:
From what I can tell, those snapshots are not ODSP snapshots. I remember looking into this when I worked on the local ODSP driver, and it was decided to keep the snapshot files in the tests.